cnf-tests/testsuites/e2esuite/dpdk/numa_node_sriov.go

```diff
@@ -152,12 +174,25 @@ func createSriovNetworkAndPolicyForNumaAffinityTest(numVFs int, intf *sriovv1.In
 		sriovclient, policyGeneratedName, namespaces.SRIOVOperator,
 		intf.Name+vfSelector, nodeName, numVFs,
 		resourceName, "netdevice",
-		withExcludeTopology(false),
+		withExcludeTopology(excludeTopology),
 	)
 	ExpectWithOffset(1, err).ToNot(HaveOccurred())
 
-	sriovnetwork.CreateSriovNetwork(sriovclient, intf, "test-numa-0-exclude-topology-false",
-		sriovnamespaces.Test, namespaces.SRIOVOperator, "testNuma0ExcludeTopoplogyFalse", ipam)
+	err = sriovnetwork.CreateSriovNetwork(sriovclient, intf, policyGeneratedName+"network",
+		sriovnamespaces.Test, namespaces.SRIOVOperator, resourceName, ipam)
 	ExpectWithOffset(1, err).ToNot(HaveOccurred())
 
 }
+
+func validateE2EICMPTraffic(pod *corev1.Pod, annotation string) {
```

Comment:
Sorry, I meant only the ExecCommand. putting the pod creation inside the Eventually loops bring to creating multiple pods.

I had in my mind something like
```suggestion
		serverPod := pods.DefinePod(sriovnamespaces.Test)
		serverPod = pods.RedefinePodWithNetwork(serverPod, annotation)
		command := []string{"bash", "-c", "ping -I net1 192.0.2.250 -c 5"}
		serverPod, err := client.Client.Pods(sriovnamespaces.Test).
			Create(context.Background(), serverPod, metav1.CreateOptions{})
		Expect(err).ToNot(HaveOccurred())
		
		Eventually(func(g Gomega) error {
     		    _, err = pods.ExecCommand(client.Client, *pod, command)
     		    return err
     		}, 30*time.Second, 1*time.Second).Should(Succeed(), "ICMP traffic failed over SRIOV interface pod interface")
```

WDYT?

Reply:
Better! :) 
Changed